packages/radio-button/package.json

```diff
@@ -18,6 +18,7 @@
   ],
   "dependencies": {
     "@hig/behaviors": "^1.1.1",
+    "@hig/spacer": "^1.0.4",
```

Comment:
This looks like it should go into `devDependencies` since it's only used in `storybook`.